docs/pages/plugins-changes-in-catalina.md

```diff
@@ -0,0 +1,96 @@
+     username: 'AC_USERNAME'
+     password: '@keychain:AC_PASSWORD'
+   ```
+
+Replace `TEAM` with the name of your team on App Store Connect.
+
+Replace the `AC_USERNAME` with your App Store Connect username (usually an email address). Because App Store Connect now requires two-factor authentication (2FA) on all accounts, you must create an app-specific password, as described in [Using app-specific passwords](https://support.apple.com/en-us/HT204397).
+
+To avoid including your password as cleartext in the file, you can provide a reference to a keychain item. This assumes the keychain holds a keychain item named `AC_PASSWORD` with an account value matching the username `AC_USERNAME`. Note that skpm can’t access your iCloud keychain for security reasons, so the item must be in your login keychain. You can add a new keychain item using the Keychain Access app, or from the command line using the security utility:
+
+```bash
+security add-generic-password -a "AC_USERNAME" -w <secret_password> -s "AC_PASSWORD"
+```
+
+### Custom command
+
+Alternatively, if you have some more complex needs for your plugin, you can replace the default skpm process by a custom bash command.
+
+In the `.skpmrc` file, add the following content:
+
+```yaml
+notarization:
+  command: './custom-script.sh'
+```
+
+Skpm will execute `./custom-script.sh ./path/to/plugin.zip` and expect the same zip to be notarized at the end.
+
+
+## Notarizing plugins using Xcode
+
+(Credits to Aby Nimbalkar for the information)
+
+Requirements for notarizing plugins using Xcode and the command line:
+
+- [Two-factor authentication](https://support.apple.com/en-us/HT204915) needs to be enabled for your account.
+- An [app-specific password](https://support.apple.com/en-us/HT204397)to use in Terminal for the notarization. It will have a format like this: `wzac-jvbe-nhqu-wjao`
+
+1. Make sure your plugin framework is signed as a Developer ID Application in Xcode:
+
+    ![](/images/developer/dev-id.png)
+
+2. Drop your framework into your .sketchplugin bundle and zip the entire bundle.
+3. Code-sign the zip using this command in the terminal. The identifier must match the bundle identifier of your plugin framework (not necessarily your plugin identifier in the manifest).
+
+    ```bash
+    codesign -f -s "Developer ID Application: John Doe" --timestamp --identifier "com.organization.PluginName" /Users/johndoe/Desktop/PluginName.zip
+    ```
+
+4. Send the zip for notarization:
+    ```bash
+    xcrun altool --notarize-app -f /Users/johndoe/Desktop/PluginName.zip --primary-bundle-id "com.organization.PluginName" -u "yourAppleIDEmail@gmail.com" -p "wzac-jvbe-nhqu-wjao"
+    ```
+
+5. Apple will then send you an email in about 3-4 minutes that your bundle is notarized. You can now release the plugin.
+6. If you make any changes to your plugin framework you’ll need to notarize again, so do all this as the last step before release.
+
+## Further reading
```

Comment:
We use _Related resources_ in [other places](https://developer.sketch.com/plugins/cocoascript) and _Next steps_, for consistency I suggest to use the one that works better.

```suggestion
## Related resources
```